src/packages/migrate/src/utils/flagErrors.ts

```diff
@@ -25,7 +25,7 @@ export class EarlyAcessFlagError extends Error {
   constructor() {
     super(
       `This feature is currently in Early Access. There may be bugs and it's not recommended to use it in production environments.
-      Please provide the ${chalk.green(
+Please provide the ${chalk.green(
```

Comment:
I guess this is a generic helper method, so ok it to keep it?

Reply:
Yes I think so